products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -685,8 +685,13 @@ public final class BuildSystem {
         info.pointee.inode = UInt64(s.st_ino)
         info.pointee.mode = UInt64(s.st_mode)
         info.pointee.size = UInt64(s.st_size)
-        info.pointee.mod_time.seconds = UInt64(s.st_mtimespec.tv_sec)
-        info.pointee.mod_time.nanoseconds = UInt64(s.st_mtimespec.tv_nsec)
+        #if os(Linux)
+            info.pointee.mod_time.seconds = UInt64(s.st_mtim.tv_sec)
+            info.pointee.mod_time.nanoseconds = UInt64(s.st_mtim.tv_nsec)
+        #else
```

Comment:
Please flip the if-else so that it's an `#if os(Darwin)` since `st_mtimespec` is the outlier here. This this will work for FreeBSD as well.

Reply:
Thanks!